articles/advisor/advisor-reference-reliability-recommendations.md

```diff
@@ -645,6 +645,23 @@ In active-active configuration, both instances of the VPN gateway establish S2S
 
 Learn more about [Virtual network gateway - VNetGatewayActiveActive (Enable Active-Active gateways for redundancy)](https://aka.ms/aa_vpnha_learnmore).
 
+### Use HEAD health probes
+
+Front Door Health probes can use either the GET or HEAD HTTP method. It’s a good practice to use the HEAD method for health probes, which reduces the amount of traffic load on your origins.
+
+Learn more about [Use HEAD health probes](https://aka.ms/afd-use-health-probes).
+
+### Use managed TLS certificates
+
+When Front Door manages your TLS certificates, it reduces your operational costs, and helps you to avoid costly outages caused by forgetting to renew a certificate. Front Door automatically issues and rotates the managed TLS certificates.
```

Comment:
Front Door management of your TLS certificates reduces your operational costs and helps you to avoid costly outages caused by forgetting to renew a certificate.